features/dd-sdk-android-trace/src/test/kotlin/com/datadog/android/trace/internal/data/OtelTraceWriterTest.kt

```diff
@@ -301,5 +307,21 @@ internal class OtelTraceWriterTest {
         verifyNoMoreInteractions(mockSdkCore, mockTracingFeatureScope)
     }
 
+    private fun createNonEmptyDdSpans(forge: Forge, isDropSampling: Boolean): MutableList<DDSpan> {
```

Comment:
maybe for the clarity we can use `includeDropSamplingPriority` here instead of `isDropSampling` or something similar?